genemachine.sh

```diff
@@ -0,0 +1,49 @@
+#therese reisch, tara neufell, and kevin buck bash project :))
+#start in the bioinformaticsProject, all of this code is assuming that our TA has the same directory structure as we do
+#usage: bash genemachine.sh
+
+#put all of the mcrA reference files into one file and all of the hsp gene reference files into another file
+touch  ref_sequences/mcrAmaster.fasta
+for index in {01..18}
+	do cat ref_sequences/mcrAgene_$index.fasta >> ref_sequences/mcrAmaster.fasta
+done
+
+touch ref_sequences/hsp70master.fasta
+for index in {01..22}
+	do cat ref_sequences/hsp70gene_$index.fasta >> ref_sequences/hsp70master.fasta
+done
+
```

Comment:
We just need one line for this
cat ref_sequences/mcrAgene_* > ref_sequences/ref_mcrAgenes.fasta
same for hsp70